docs/content/v0.20/packaging.md

```diff
@@ -22,17 +22,17 @@ name --- that is, ends with `github.com/kubestellar/kubestellar/`.
 
 Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub. A contributor can use the individual `make` commands mentioned below to update subsets, or use the following command to update them all.
 
-```
+```shell
 make all-generated
 ```
 
 The following command, checked in CI, checks whether all the derived files have been correctly derived. It must be invoked in a state where the `git status` is clean, or at least the dirty files are irrelevant; the current commit is what is checked.
 
-```
+```shell
 hack/verify-codegen.sh
 ```
 
-## Files generated by controller-gen
+### Files generated by controller-gen
```

Comment:
I think the heading level of this should be 4, the same as 'Files generated by code-generator', since they both are under `### Derived files`.

More importantly, line 41 `make generated` ->`make generate`. GH UI doesn't allow me to comment there so I'm commenting here instead.

Reply:
Thanks, fixed.